apps/ledger-live-desktop/src/renderer/storage.ts

```diff
@@ -52,6 +55,7 @@ type DatabaseValues = {
   countervalues: Countervalues;
   postOnboarding: PostOnboarding;
   settings: Settings;
+  trustchainStore: TrustchainStore;
```

Comment:
correct me if i'm wrong but i think we didn't implemented the encryption of the trustchainStore yet? 
i can help on this part

Reply:
Not yet